iquip/apps/explorer.py

```diff
@@ -69,34 +68,34 @@ def __init__(
 
     def run(self):
         """Overridden.
-        
+
         Fetches the file list using a command line.
 
         Searches for only files in path, not in deeper path and adds them into the widget.
         After finished, the fetched signal is emitted.
         """
-        experimentList = cmdtools.run_command(f"artiq_client ls {self.path}").stdout
-        experimentList = experimentList.split("\n")[:-1]  # The last one is always an empty string.
+        try:
+            response = requests.get("http://127.0.0.1:8000/ls/",
+                                    params={"directory": self.path},
+                                    timeout=10)
+            response.raise_for_status()
+            experimentList = response.json()
+        except requests.exceptions.RequestException:
+            pass
```

Comment:
Even though we are not logging by now, but let's temporarily print something about the exception information.